buddyboss/buddyboss-platform
# Added caching functions to prevent the repeated SQL queries

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2160 .

### How to test the changes in this Pull Request:

1. Install Query Monitor Plugin
2. Make sure the Moderation component active with Platform, Pro, and Theme.
3. Go to Frontend pages Homepage, Members, News Feed.
4. Most of these pages trigger the same SQL queries a bunch of times.

### Proof Screenshots or Video
https://www.loom.com/share/9d9749cdec8c45f193da78149bd5dc4c



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
